.goreleaser.yml

```diff
@@ -20,6 +20,6 @@ dockers:
   - goos: linux
     goarch: amd64
     image_templates:
-      - "jamf/sonarqube-pr-issues:latest"
```

Comment:
not sure about this so I changed it back for now 🤔  because if we want to merge this to master branch, then create PR to upstream, change will be from master to master upstream, but we don't want this change to push through